azure-stack/hci/manage/manage-secure-baseline.md

```diff
@@ -0,0 +1,117 @@
+
+1. Connect to your Azure Stack HCI node.
+1. Run the following cmdlet using local administrator credentials or deployment user account credentials.
+
+## Configure security during deployment
+
+When deploying your cluster via the Supplemental Package, you can modify drift control settings and other security settings that constitute the security baseline. Changes you make to security settings are also reflected in the *config.json* file that you create using the deployment tool.
+
+The following table describes security settings that can be configured on your Azure Stack HCI cluster during deployment.
+
+| Feature area | Feature     |Description           | Supports drift control? |
+|--------------|-------------|----------------------|---------------------------------|
+| Governance                 | [Security baseline](.././concepts/secure-baseline.md)            | Maintains the security defaults on each server. Helps protect against changes.  | Yes                             |
+| Credential protection      | [Windows Defender Credential Guard](/windows/security/identity-protection/credential-guard/credential-guard)     | Uses virtualization-based security to isolate secrets from credential-theft attacks. | Yes                             |
+| Application control        | [Windows Defender Application control](/windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview#windows-defender-application-control)           | Controls which drivers and apps are allowed to run directly on each server.           | No                              |
+| Data at-rest encryption    | [BitLocker for OS boot volume](/windows/security/information-protection/bitlocker/bitlocker-overview)          | Encrypts the OS startup volume on each server.                                        | No                              |
+| Data at-rest encryption    | [BitLocker for data volumes](/windows/security/information-protection/bitlocker/bitlocker-overview)            | Encrypts cluster shared volumes (CSVs) on this cluster                               | No                              |
+| Data in-transit protection | [Signing for external SMB traffic](/troubleshoot/windows-server/networking/overview-server-message-block-signing)      | Signs SMB traffic between this system and others to help prevent relay attacks.       | Yes                             |
+| Data in-transit protection | [SMB Encryption for in-cluster traffic](/windows-server/storage/file-server/smb-security#smb-encryption) | Encrypts traffic between servers in the cluster (on your storage network).            | No                              |
+
+### Modify security after deployment
+
+Once the deployment is complete, you can modify security features while maintaining drift control. Some features require a reboot to take effect.
+
+### PowerShell cmdlet properties
+
+The following cmdlet properties are for the *AzureStackOSConfigAgent* module.
+
```

Comment:
Check with Carlos if the module is automatically installed during the deployment on the Azure Stack HCI cluster or the user has to import it.

Also add a tie in sentence such as: To modify security feature, you will use PowerShell cmdlets.  During the deployment, the AzureStackOSConfigAgent module is installed.

Reply:
Done > @Carlos confirmed that the module is installed during deployment,